bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/RuntimeUtils.java

```diff
@@ -174,6 +175,8 @@ public static void handleUsageError(String errorMsg) {
 
     public static void logBadSad(Throwable throwable) {
         // These errors are unhandled errors in JVM, hence logging them to bre log.
+        errStream.println(INTERNAL_ERROR_MESSAGE);
+        errStream.println();
```

Comment:
Do we need this new line here since we already have a `\n` part in the error message?